### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_Deploy Web App_Screen Reader: NVDA is not reading the list item as ‘Selected’ 

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3352]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #Screen Reader; #C+E-H; #Java tooling for IntelliJ/Eclipse; #NVDA

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
Screen Reader: NVDA
MAS Violated: MAS1.3.1

**Note: Similar issue is found for 'Create Service principle Status' popup**

**Repro Steps:** 
1) Start NVDA and Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace settings/preferences popup
3) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
4) Tab till File-->New-->Project--->Dynamic Web Project
5) Within Eclipse's Project Explorer view, expand Newly created dynamic web project. Right-click WebContent, click New-->JSP File
6) Tab till Next button and press enter
7) Select New JSP File (html) and press Finish
8) Open JSP file and write any code in the editor
9) In Project Explorer view, right-click your project, choose Azure-->Publish as Azure Web App
10) Tab till the list items and verify if NVDA is reading as selected

**Actual Result:** 
While navigating to the list items NVDA is not reading that the list is selected or not

**Expected Result:** 
When any list item is selected NVDA should read as selected

**User Impact:** 
Screen Reader users will not know that the list item is selected if NVDA does not read as selected 

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

![Bug_23_NVDA is not reading selected for the list items](https://user-images.githubusercontent.com/41109135/62207072-d626e500-b3b0-11e9-955d-14a12415cea7.png)

**Comment by Flanker32** [comment]
@Tanushagottipati This is a common behavior of Eclipse Table without checkbox, here is a gif of Eclipse Preferences, NVDA will not read the items as selected either.

![3352](https://user-images.githubusercontent.com/12445236/64519597-fdb18b80-d326-11e9-9b51-2867b7ab314b.gif)

**Comment by Flanker32** [comment]
* Importance of experience: Little, it is a nice to have feature read items in table as selected
* Quality of experience: Little, NVDA do read the selected items, and it okay to regard the last read item as the selected one.
* Alternate solutions: No alternate technologies or options can be leveraged.
* Evidence: All evidence is provided in this issue.
* [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Flanker32** [comment]
@Tanushagottipati This is an external issue of Eclipse and we have already linked it to AAT, please help us verify it, thanks!

**Comment by Tanushagottipati** [comment]
External Bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=550892

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749752
